### PR TITLE
Return semantic error in validator

### PR DESCRIPTION
**Pull request by yixinglu**
as title.

### CODE REVIEW

**Review by laura-ding — DISMISSED** [review]

File: src/validator/GetSubgraphValidator.cpp [hunk 1]
```diff
@@ -22,29 +22,19 @@ Status GetSubgraphValidator::validateImpl() {
     auto* gsSentence = static_cast<GetSubgraphSentence*>(sentence_);
 
     status = validateStep(gsSentence->step(), steps_);
```
Comment (Shylock-Hg) [inline src/validator/GetSubgraphValidator.cpp:24]: The `status` is useless.

Comment (yixinglu) [reply src/validator/GetSubgraphValidator.cpp]: fixed

File: src/validator/GetSubgraphValidator.cpp [hunk 2]
```diff
@@ -21,30 +21,11 @@ Status GetSubgraphValidator::validateImpl() {
     Status status;
```
Comment (Shylock-Hg) [inline src/validator/GetSubgraphValidator.cpp:21]: Delete this define.

Comment (yixinglu) [reply src/validator/GetSubgraphValidator.cpp]: Good catch!

**Review by CPWstatic — APPROVED** [review]
Thanks.

**Review by jievince — APPROVED** [review]

**Pull request merged**